test/integration/src/test/java/io/pravega/test/integration/ReaderGroupNotificationTest.java

```diff
@@ -106,26 +125,31 @@ public void setUp() throws Exception {
 
     @After
     public void tearDown() throws Exception {
-        executor.shutdownNow();
+        executorService().shutdownNow();
```

Comment:
This is not needed. If you look at the base class, it is shut down for you using the proper shutdown sequence.